gapis/api/vulkan/custom_replay.go

```diff
@@ -452,3 +453,7 @@ func (a *ReplayAllocateImageMemory) Mutate(ctx context.Context, id api.CmdID, s
 	a.PMemory.Slice(0, 1, l).Write(ctx, []VkDeviceMemory{memory}, a, s, b)
 	return err
 }
+
+func (i AllocationCallbacks) value(b *builder.Builder, cmd api.Cmd, s *api.GlobalState) value.Value {
+	return value.AbsolutePointer(0)
```

Comment:
Please comment why you're always using a 0.

Reply:
Done